src/main/java/com/cdancy/bitbucket/rest/domain/sshkey/AccessKey.java

```diff
@@ -17,16 +17,46 @@
 
 package com.cdancy.bitbucket.rest.domain.sshkey;
 
+import java.util.List;
+
 import com.google.auto.value.AutoValue;
+import org.jclouds.javax.annotation.Nullable;
 import org.jclouds.json.SerializedNames;
 
+import com.cdancy.bitbucket.rest.domain.common.Error;
+import com.cdancy.bitbucket.rest.domain.common.ErrorsHolder;
+import com.cdancy.bitbucket.rest.domain.repository.Repository;
+import com.cdancy.bitbucket.rest.domain.project.Project;
+import com.cdancy.bitbucket.rest.BitbucketUtils;
+
 @AutoValue
-public abstract class AccessKey {
+public abstract class AccessKey implements ErrorsHolder {
```

Comment:
:bangbang: The class name 'AccessKey' doesn't match 'Abstract[A-Z][a-zA-Z0-9]*' (#QR73:33#)-PMD_ClassNamingConventions